terraform/modules/os_monitoring/cloudwatch.tf

```diff
@@ -0,0 +1,55 @@
+resource "aws_cloudwatch_metric_alarm" "high_memory" {
+  alarm_name                = "high-memory-${var.component}"
+
+  # https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/mon-scripts.html#using_put_script
+  metric_name               = "MemoryUtilization"
+  namespace                 = "System/Linux"
+
+  comparison_operator       = "GreaterThanOrEqualToThreshold"
+  evaluation_periods        = "2"
+  period                    = "300"
+  statistic                 = "Average"
+  threshold                 = "${var.high_memory_threshold}"
+  alarm_description         = "This metric monitors ec2 memory utilization for the instance ${var.instance_id}"
+  alarm_actions = ["${var.sns_topic_arn}"]
+
+  dimensions {
+    InstanceId = "${var.instance_id}"
+  }
+}
+
+resource "aws_cloudwatch_metric_alarm" "high_cpu" {
+  alarm_name                = "high-cpu-${var.component}"
+  comparison_operator       = "GreaterThanOrEqualToThreshold"
+  evaluation_periods        = "2"
+  metric_name               = "CPUUtilization"
+  namespace                 = "AWS/EC2"
+  period                    = "120"
```

Comment:
Not a blocker: Pretty sure I just copied-and-pasted `period` and `evaluation_periods` values here. Should we parameterize? Does it make sense to keep them different?

Reply:
Yeah, we should parameterize all of that, but as this branch has gotten huge, let's do another branch on that after closing this PR.